package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "koa-body",
-  "version": "1.5.0",
+  "version": "1.6.0",
```

Comment:
I think i should not update this one when contributing. It's maintainer's thing and part of the flow.